boards/arm/rddrone_fmuk66/doc/index.rst

```diff
@@ -6,15 +6,37 @@ NXP RDDRONE-FMUK66
 Overview
 ********
 
-The RDDRONE FMUK66 is an drone control board with commonly used peripheral
-connectors and a Kinetis K66 on board.
+RDDRONE FMUK66 is primarily designed as an autonomous drone 
+"Flight Management Unit" control board which incorporates DroneCode standard
+common peripheral connectors. The MCU onboard is a Kinetis K66. 
+
+The FMUK66 hardware includes an onboard IMU (Intertial measurement units),
+ports for off-the-shelf GPS modules with integrated arming swtiches, 
+LEDs and enunciators, SDCARD for logging, CAN ports, PWM ports, 
+100BaseT1 "2-wire" Ethernet, and SE050 Secure Element. 
+
+Note that historically this board is supported by another RTOS, NuttX, 
+and the PX4 flight stack running on top, and can fly Multicopters, VTOL and 
+traditional aircraft, ground based rovers and water going vessels. 
+The intention here is to have a functional port to Zephyr RTOS so that
+experimentation and development of flight/Vehicle stacks can also begin 
+with Zephyr.
+The FMUK66 hardware is suitable for use in other mobile robotics applications,
+or as hardware where where CAN, T1 Ethernet, or Secure development is needed.
+
+- The RDDRONE-FMUK66 kit comes with a J-Link Edu Mini for programming and UART
+ console as well as a power distribution and measuring module as well as a GPS.
 
```

Comment:
did you intend for this to be a bullet item of one? Sphinx build is complaining about this:

":Bullet list ends without a blank line; unexpected unindent."